### ISSUE TITLE
Can't append a new key with hook

### ISSUE THREAD

**Issue by seqizz** [qtile/qtile#1496]
Hi there :wave: 

First of all, thanks for this great WM. I'm currently building my workflow.

I wanted to add a hook, so every new "undefined" new dynamic group is added to one of my GroupBox widgets and have a shortcut assigned.
I've used `@hook.subscribe.addgroup`.I can see new group under GroupBox, but the shortcut key assignment didn't work. I've logged all the elements and all seem to be the correct type, and I don't get any errors with appending.

tldr; Can I add a key dynamically under a hook function later? Or is it too late?

**Comment by pawamoy** [comment]
Hello! Welcome!

Could you show us the code you wrote to dynamically add a shortcut? I'm guessing you are simply appending an element to the `keys` list?

**Comment by seqizz** [comment]
Yeah, I just added to the list. But I realized the `qtile` object sent has a keyMap in it, I'll try to manipulate that and report back if I can manage.

**Comment by seqizz** [comment]
Closing since I couldn't manage and realized qtile is not for me. Thanks anyway :crossed_fingers:
